src/index_runner/es_indexers/annotated_metagenome_assembly.py

```diff
@@ -54,16 +53,17 @@ def _index_ama(features_file_gz_path, data, ama_id, ver_ama_id):
     }
     ama_index['id'] = ama_id
     yield ama_index
-    ama_index['id'] = ver_ama_id
-    ama_index['index'] = _VER_AMA_INDEX_NAME
-    yield ama_index
+    ver_ama_index = dict(ama_index)
+    ver_ama_index['id'] = ver_ama_id
+    ver_ama_index['index'] = _VER_AMA_INDEX_NAME
+    yield ver_ama_index
 
     if config()['skip_features']:
         # Indexing of AMA features is turned off in the env
         return
 
     # unzip gzip file.
-    features_file_path = _DIR + "/features.json"
+    features_file_path = tmp_dir + "/" + ver_ama_id.replace(':', "_") + ".json"
```

Comment:
just do `os.path.join(tmp_dir, ver_ama_id.replace(':', '_') + '.json')`. It has some other minor advantages such as cross-OS compatibility